lib/config/app_config.dart

```diff
@@ -24,6 +24,8 @@ abstract class AppConfig {
   static String _privacyUrl = 'https://twake.app/en/privacy/';
 
   static String get privacyUrl => _privacyUrl;
+  static String get registationUrl =>
+      'https://register.tom-dev.xyz/?post_registered_redirect_url=twake.chat://registered';
```

Comment:
If it is a different home-server, will the url change?

Reply:
done. using config.json now